lib/travis/cli/repo_command.rb

```diff
@@ -83,6 +82,16 @@ def detect_slug
           end
         end
 
+        def parse_remote(url)
+          if url.start_with?('git@github.com:')
```

Comment:
I just noticed that this assumes that the remote repo resides on github.com. This is true for the most part, but we also have Travis Enterprise, which works with GitHub Enterprise. From what I understand, the only thing we know for certain is that the user name is `git`.

Perhaps
```ruby
if url =~ /^git@[^:]+:/
⋮
```
is reasonably close (if not completely accurate).